cron/update_s3_htmls.sh

```diff
@@ -9,7 +9,7 @@ set -e
 
 # Upload for all CUDA/cpu versions if not given one to use
 if [[ -z "$CUDA_VERSIONS" ]]; then
-    export CUDA_VERSIONS=('cpu' 'cu92' 'cu100' 'cu101' 'cu102' 'cu110' 'rocm3.7')
+    export CUDA_VERSIONS=('cpu' 'cu92' 'cu100' 'cu101' 'cu102' 'cu110' 'rocm3.7' 'rocm3.8')
```

Comment:
@seemethere should we add this together with the nightly PR or should we add it AFTER the files are being uploaded?

Reply:
it doesn't actually matter, this will work even if there are no `rocm3.8` packages available